dask_geopandas/core.py

```diff
@@ -88,7 +92,19 @@ def crs(self):
     @crs.setter
     def crs(self, value):
         """Sets the value of the crs"""
-        raise NotImplementedError
+        self.dask = self.set_crs(value).dask
+
+    def set_crs(self, value):
+        self._meta.crs = value
+
+        def set_crs(df, crs):
+            df = df.copy(deep=False)
+            df.crs = crs
+            return df
```

Comment:
This can be changed later, but we typically avoid using dynamically created functions like this, and instead try to use top level functions.  This is useful because dynamically generated functions are a bit more expensive and complex to serialize.

Reply:
Ah ok I'd seen that pattern and just hadn't thought about the rationale. I'll move it out.